.gitignore

```diff
@@ -1,4 +1,9 @@
-*.swp
-_build
-syslog-message.install
+# Dune generated files
+_build/
+*.install
+
+# Merlin configuring file for Vim and Emacs
 .merlin
```

Comment:
Modern `dune` doesn't generate `.merlin` files anymore, instead it serves the config via `dune-ocaml-merlin` directly. But I wasn't able to quickly discover when that change happened.

Similarly I don't think it generates `.install` files anymore, at least not at the default target.

Reply:
Thanks for your comment. I'm going to check the dune docs again. Excluding .install might only be necessary if dune also generates the opam file.